app/controllers/custom_tips_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CustomTipsController < SecureController
+  def create
+    create_custom_tip
+  rescue ActiveRecord::RecordInvalid, ActiveRecord::RecordNotUnique
```

Comment:
Why even try `update_custom_tip` on `RecordInvalid`?

Reply:
This was copied from custom pages, I can remove it